src/main/scanLogic/scanRunners/binaryRunner.ts

```diff
@@ -353,7 +360,7 @@ export abstract class BinaryRunner {
         // 1. Save requests as yaml file in folder
         fs.writeFileSync(requestPath, request);
         // 2. Run the binary
-        await this.runBinary(requestPath, path.dirname(requestPath), checkCancel).catch(error => {
+        await this.runBinary(requestPath, this._verbose ? undefined : path.dirname(requestPath), checkCancel).catch(error => {
             if (error.code) {
```

Comment:
The older .then syntax should be replaced with the more modern and consistent async/await syntax. Please consider refactoring the code accordingly.